invenio_users_resources/records/mappings/os-v1/users/user-v1.0.0.json

```diff
@@ -69,6 +69,14 @@
           },
           "email_visibility": {
             "type": "keyword"
+          },
+          "locale": {
+            "type": "keyword",
+            "index":"false"
```

Comment:
@slint I was chatting with Martin about this, do you think it will be necessary to index the locale and timezone? To me, we probably will not search by these fields...
Another way could be that we add:
```
"dynamic": true
```
as done for the `profile` above.

Reply:
I agree, `dynamic: true` is probably the best in this case, since we don't know what other preferences an instance might want to store additionally. This still makes the fields searchable though, but they are protected by the query transformer.

We should then also add the field to the `dynamic_tempaltes` above like so:


```python
    "dynamic_templates": [
        {
          "profile": {
            "path_match": "profile.*",
            "mapping": {
              "type": "keyword"
            }
          },
          "preferences": {
            "path_match": "preferences.*",
            "mapping": {
              "type": "keyword"
            }
          }
        }
    ],
```

Reply:
if i add this it crashes during setup 

`'mapper_parsing_exception', 'A dynamic template must be defined with a name'`

Reply:
Let's discuss this error IRL.

Reply:
out of curiosity, what was the source of the error?

Reply:
It was something stupid :sweat: 

```json
    "dynamic_templates": [
      {
        "profile": {
          "path_match": "profile.*",
          "mapping": {
            "type": "keyword"
          }
        }
      },
      {
        "preferences": {
          "path_match": "preferences.*",
          "mapping": {
            "type": "keyword"
          }
        }
      }
    ],
```

instead of
```json
    "dynamic_templates": [
        {
          "profile": {
            "path_match": "profile.*",
            "mapping": {
              "type": "keyword"
            }
          },
          "preferences": {
            "path_match": "preferences.*",
            "mapping": {
              "type": "keyword"
            }
          }
        }
    ],
```

IE one dict per mapping, and not one dict with 2 dicts inside